Library/Homebrew/github_packages.rb

```diff
@@ -109,16 +109,16 @@ def self.root_url(org, repo, prefix = URL_PREFIX)
     # docker/skopeo insist on lowercase org ("repository name")
     org = org.downcase
 
-    "#{prefix}#{org}/#{repo_without_prefix(repo)}"
+    "#{prefix}/#{org}/#{repo_without_prefix(repo)}"
   end
 
   def self.root_url_if_match(url)
     return if url.blank?
 
-    _, org, repo, = *url.to_s.match(URL_REGEX)
-    return if org.blank? || repo.blank?
+    _, prefix, org, repo, = *url.to_s.match(URL_REGEX)
+    return if prefix.blank? || org.blank? || repo.blank?
 
-    root_url(org, repo)
+    root_url(org, repo, prefix)
```

Comment:
Is `root_url` ever used in a two-argument form now?

Reply:
Only here, so far as I could find.

Reply:
Ok, I think the default argument can be removed from it then 👍🏻